docs/concepts.rst

```diff
@@ -0,0 +1,462 @@
+
+.. _`Concepts`:
+
+Concepts
+========
+
+
+TODO: We need a section on profiles
```

Comment:
Add a section "Management profiles" at the end, and have the TODO in that section.